### PR TITLE
Show extra info to users who are not registered for minting

### PR DESCRIPTION
**Pull request by shelbyd**
Also add a button to allow unregistered users to try to register.

### CODE REVIEW

File: src/popup/components/Protocol/DataScreen/Minting.tsx [hunk 1]
```diff
@@ -42,22 +49,103 @@ export function Minting() {
...
+    .map((e) => <MintingError error={e} />)
+    .unwrapOrDefault(<p>Loading</p>);
+
+  return (
+    <NotRegisteredContainer>
+      <div className="text" data-tip data-for="notRegistered">
+        <Activated text="Not Registered" icon={IconNames.INVALID} />
+      </div>
+
+      <ReactTooltip id="notRegistered" place="top" effect="solid">
```
Comment (melatron) [inline src/popup/components/Protocol/DataScreen/Minting.tsx:103]: We can maybe get the effect of top left with offset. Check this thread - https://stackoverflow.com/questions/59826015/how-to-change-position-of-reacttooltip

Comment (shelbyd) [reply src/popup/components/Protocol/DataScreen/Minting.tsx:103]: Possibly, but I don't really trust the position of the library. Even with "top" it is showing up to the left. So I'm okay with letting it be wherever it's going to be. Adding an offset just increases the chance it shows up somewhere stupid.

File: src/services/MintingRegistrar.ts [hunk 2]
```diff
@@ -13,44 +13,47 @@ export class MintingRegistrar {
...
-          console.log("Not registered for minting, trying to register");
-          try {
+      await withLock(this.storage, this.key("lock"), async () => {
+        try {
+          const protocol = await getProtocolService();
+          await protocol.ensureIdentityRegistered();
+          const isRegistered = await protocol.isRegisteredForMinting();
+          if (isRegistered) {
+            console.log(
+              "Already registered for next minting, not doing anything",
```
Comment (melatron) [inline src/services/MintingRegistrar.ts:37]: Remove the comma from the `console.log`

**Review by melatron — APPROVED** [review]

File: src/services/MintingRegistrar.ts (hunk 2 above)
Comment (shelbyd) [reply src/services/MintingRegistrar.ts:37]: This came from the formatter. So it would thrash or fail the build if formatting doesn't match.

**Pull request merged**